templates/blank/src/app/(frontend)/page.tsx

```diff
@@ -0,0 +1,55 @@
+import { headers as getHeaders } from 'next/headers.js'
+import Image from 'next/image'
+import { getPayload } from 'payload'
+import React from 'react'
+import { fileURLToPath } from 'url'
+
+import config from '@/payload.config'
+import './styles.css'
+
+export default async function HomePage() {
+  const headers = await getHeaders()
+  const payloadConfig = await config
+  const payload = await getPayload({ config: payloadConfig })
+  const { user } = await payload.auth({ headers })
+
+  const adminRoute = payloadConfig.routes?.admin || '/admin'
```

Comment:
No need to fallback to `/admin` here. We sanitize the config and guarantee this field is populated. Should just write this inline in the JSX.